ReviewMonitor/Model/Meta.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Meta.swift
+//  ReviewMonitor
+//
+//  Created by Eliott Hauteclair on 3/01/18.
+//  Copyright © 2018 Tayal, Rishabh. All rights reserved.
+//
+
+import UIKit
+
+class Meta: NSObject {
+    var name = String()
+    var version = String()
+    var copyright = String()
+    var status = String()
+    var languages = String()
+    var keywords = String()
+    var support = String()
+    var marketing = String()
+    var available = String()
+    var watchos = String()
+    var beta = String()
+    var bundleID = String()
+    var primaryCateg = String()
+    var primaryCategSub1 = String()
+    var primaryCategSub2 = String()
+    var secondaryCateg = String()
+    var secondaryCategSub1 = String()
+    var secondaryCategSub2 = String()
+
+    func set(_ name: String, version: String, copyright: String, status: String, languages: String, keywords: String, support: String, marketing: String, available: String, watchos: String, beta: String, bundleId: String, primaryCateg: String, primaryCategSub1: String, primaryCategSub2: String, secondaryCateg: String, secondaryCategSub1: String, secondaryCategSub2: String) {
```

Comment:
Instead of `set`, let's use `init` method.

Reply:
How?

Reply:
Refer to this: https://github.com/RishabhTayal/ReviewMonitor/blob/a26e509f69f1e9875a58e863745fe953306acaf5/ReviewMonitor/Model/App.swift#L18. Take `dictionary` as parameter instead of taking and the values individually.

Reply:
But how can I do that with this code? 
                    self.meta.set(self.app.name, version: ver, copyright: copyright, status: status, languages: lng, keywords: keyw, support: supportUrl, marketing: marketingUrl, available: avail, watchos: watchos, beta: beta, bundleId: self.app.bundleId, primaryCateg: primarycat, primaryCategSub1: primarySubcat, primaryCategSub2: primarySubSeccat, secondaryCateg: secondarycat, secondaryCategSub1: secondarySubcat, secondaryCategSub2: secondarySubSeccat)